### PR TITLE
Feature/navbar sign up

### PR DESCRIPTION
**Pull request by chennan47**
<b>Purpose</b>
Add sign up button to navbar. Solve https://github.com/CenterForOpenScience/osf.io/issues/2469.

<b>Changes</b>
before:
screen shot 2015-04-18 at 4 08 05 pm

after:
screen shot 2015-04-18 at 4 08 43 pm

### CODE REVIEW

**Comment by chennan47** [comment]
@sloria rdy for review

File: website/static/css/site.css [hunk 1]
```diff
@@ -1856,6 +1856,14 @@ span#profileFullname{
     margin-bottom: 2px;
     margin-left: 0;
     margin-right: 0;
+    padding-right:3px;
+}
+
+.navbar-form a.btn-sign-up{
```
Comment (sloria) [inline website/static/css/site.css]: Is there any way we can take of bootstrap's CSS rather than having to add this custom CSS?

**Comment by sloria** [comment]
Pass finished. :point_up:

**Comment by chennan47** [comment]
Discuss with @sloria @jlcohoon @saradbowman @GaryKriebel , change the sign up button to link.
screen shot 2015-04-23 at 9 44 10 am

**Comment by GaryKriebel** [comment]
Only question I have is if the "Sign Up" and "Forgot Password" are the same font size ... they look a bit different.

**Comment by chennan47** [comment]
Yes, they are the same font. Just change the word from "Forget Password" to "Sign Up" to give a sense for comparison.
screen shot 2015-04-23 at 9 57 33 am

**Comment by JeffSpies** [comment]
Why do we need forgot password in the toolbar?

**Comment by GaryKriebel** [comment]
Would it be apparent to a user that has no idea what their password is to enter anything in the sign in box, submit, and then get to a screen where there might be a link to reset their password?

**Comment by sloria** [comment]
I think "Forgot Password" can be removed. We can just show it on the Log-in page.

**Comment by chennan47** [comment]
@sloria Removed the "Forget Password?" in 7b50373.
New Look:
screen shot 2015-04-23 at 10 32 36 am

**Comment by saradbowman** [comment]
Looking at other sites (gmail, facebook, twitter, github), wherever there
is a place to login, there is also a "forgot password?" or "need help
logging in?" link. I think there should be something obvious for users to
click on if they can't remember their passwords, at the location for
logging in-- not necessarily "forgot password?" but could be something like
a question mark next to the sign in, with a tooltip that asks if a user has
forgotten his/her password? It's not clear to me how users will get to the
log-in page.

**Comment by lbanner** [comment]
I would agree with @saradbowman 'Forgot Password?' is actually in the nav bar on Production now -- and, as a user, without it, I wouldn't see any other direction on solving a log-in problem on the homepage.

**Comment by samanehsan** [comment]
@chennan47, can you bring this up to date with develop? After that, I think this should be good to merge @sloria! :sunny:

**Comment by chennan47** [comment]
@sloria @samanehsan done updating.

**Comment by sloria** [comment]
The sign-in page has been redone since the last discussion on this. 

As a user, if I mistype my password in the navbar, I am taken to the sign-in page, which does include a "Forgot your Password?" link. See https://accounts.osf.io/login?service=https://osf.io/.

@saradbowman @lbanner Given that, do you still prefer to have "Forgot Password?"  be in the navbar?

**Comment by saradbowman** [comment]
I prefer "forgot password" or a "?" with a "forgot password?" or "trouble logging in?" tooltip. As a user, I expect a way to recover my password from the place where I'm logging in - I don't expect to have to guess at a password and recover the password from the 'login failed' page.

**Comment by JeffSpies** [comment]
What do other sites do with login in the nav bar?

**Comment by samanehsan** [comment]
These two put the "forgot password" link in the nav bar next to the password input. 

LinkedIn: 
screen shot 2015-06-10 at 10 11 56 am

Facebook:
screen shot 2015-06-10 at 10 12 06 am

**Comment by saradbowman** [comment]
Twitter, Wells Fargo, Chase, Bank of America are the same way (but login from the homepage, not from the nav bar). 

Instagram, Ally bank do "log in" button from homepage, which takes you to a separate log in page to enter username and password. 

Summary: Forgot password? appears next to the form to enter your password

**Comment by JeffSpies** [comment]
I like LinkedIn version.

**Comment by lbanner** [comment]
Me too on the LinkedIn example - incorporated very simply, but an obvious link.

**Comment by sloria** [comment]
This will be added as part of the navbar redesign, so closing this for now.

**Pull request closed without merging**